docs/supported-features.md

```diff
@@ -0,0 +1,288 @@
+# Supported Features
+
+## Browsers
+
+Any browser that supports [MSE] (media source extensions). See
+https://caniuse.com/#feat=mediasource
+
+Note that browsers with native HLS support may play content with the native player, unless
+the [overrideNative] option is used. Some notable browsers with native HLS players are:
+
+* Safari (macOS and iOS)
+* Chrome Android
+* Firefox Android
+
+However, due to the limited features offered by some of the native players, the only
+browser on which VHS defaults to using the native player is Safari (macOS and iOS).
+
+## Streaming Formats and Media Types
+
+### Streaming Formats
+
+VHS aims to be mostly streaming format agnostic. So long as the manifest can be parsed to
+a common JSON representation, VHS should be able to play it. However, due to some large
+differences between the major streaming formats (HLS and DASH), some format specific code
+is included in VHS. If you have another format you would like supported, please reach out
+to us (e.g., file an issue).
+
+* [HLS] (HTTP Live Streaming)
+* [MPEG-DASH] (Dynamic Adaptive Streaming over HTTP)
+
+### Media Container Formats
+
+* [TS] (MPEG Transport Stream)
+* [MP4] (MPEG-4 Part 14: MP4, M4A, M4V, M4S, MPA), ISOBMFF
+* [AAC] (Advanced Audio Coding)
+
+### Codecs
+
+If the content is packaged in an [MP4] container, then any codec supported by the browser
+is supported. If the content is packaged in a [TS] container, then the codec must be
+supported by [the transmuxer]. The following codecs are supported by the transmuxer:
```

Comment:
huh since when did shortlinks like this didn't need the extra brackets?? I've always done `[the transmuxer][]` or `[long name][short name]`